.azure-pipelines/pr_test_scripts.yaml

```diff
@@ -14,11 +14,6 @@ t0:
   - bgp/test_bgpmon.py
   - cacl/test_cacl_application.py
   - cacl/test_cacl_function.py
-  - console/test_console_availability.py
-  - console/test_console_driver.py
-  - console/test_console_loopback.py
-  - console/test_console_reversessh.py
-  - console/test_console_udevrule.py
```

Comment:
Why remove these scripts from the list? Are they unstable?

Reply:
As you suggested, I add only one case as the placeholader.